tests/unit_tests/epee_utils.cpp

```diff
@@ -1859,47 +1859,39 @@ TEST(parsing, strtoul)
   const char* p;
   const char* endp;
 
-  errno = 0; // Some libc's only set errno on failure, some set it to 0 on success
-
   p = "0";
   endp = nullptr;
   ul = std::strtoul(p, const_cast<char**>(&endp), 10);
-  EXPECT_EQ(0, errno);
   EXPECT_EQ(0, ul);
   EXPECT_EQ(p + 1, endp);
 
   p = "000000";
   endp = nullptr;
   ul = std::strtoul(p, const_cast<char**>(&endp), 10);
-  EXPECT_EQ(0, errno);
   EXPECT_EQ(0, ul);
   EXPECT_EQ(p + 6, endp);
 
   p = "1";
   endp = nullptr;
   ul = std::strtoul(p, const_cast<char**>(&endp), 10);
-  EXPECT_EQ(0, errno);
   EXPECT_EQ(1, ul);
   EXPECT_EQ(p + 1, endp);
 
   p = "0q";
   endp = nullptr;
   ul = std::strtoul(p, const_cast<char**>(&endp), 10);
-  EXPECT_EQ(0, errno);
   EXPECT_EQ(0, ul);
   EXPECT_EQ(p + 1, endp);
 
   p = "    \t   0";
   endp = nullptr;
   ul = std::strtoul(p, const_cast<char**>(&endp), 10);
-  EXPECT_EQ(0, errno);
   EXPECT_EQ(0, ul);
   EXPECT_EQ(p + 9, endp);
 
   p = "q";
```

Comment:
This part of the test would be the most likely to break across systems: Per the man page, some systems report EINVAL here: "EINVAL (not in C99) The given base contains an unsupported value." The c++ function makes no mention of EINVAL, so perhaps whoever wrote this test suite wanted a way to determine the actual behavior in a given build.

Reply:
I think you meant to put this comment on the test immediately below this one, yeah? According to the C standard, `strtoul` should skip over whitespace. But yes, I agree with the general point, so I updated the commit to just remove the `EXPECT_EQ(0)` from the invalid character test.